docs/user/user/howto/real-time-events/real-time-events.md

```diff
@@ -0,0 +1,37 @@
+# "Real-Time" Events
+
+## Defining "Real-Time" Glean Events
+
+"Real-Time" in the context of Glean [Events](../../../reference/metrics/event.md) directly relates to configuring Glean to send each event in an [Events Ping](../../pings/events.md) as soon as it is recorded.
+
+## Configuring Glean For "Real-Time" Events
+
+Glean event ping submission can either be configured at [initialization](../../../reference/general/initializing.md) or through [Server Knobs](../../../user/server-knobs/other/max-events.md).
+
+Setting the maximum event threshold to a value of `1` will configure Glean to submit an Events Ping for every event recorded.
+
+## Considerations
+
+### What "Real-Time" Glean Events Are _Not_
+
+Configuring Glean to submit events as soon as they are recorded does not mean to imply that the event data is available for analysis in
+real-time. There are networks to traverse, ingestion pipelines, etl, etc. that are all factors to keep in mind when considering how soon
```

Comment:
To me the hyphenated `real-time` is an adjective, the noun is `real time`. So while there are "real-time events", those are "events sent in real time".
```suggestion
real time. There are networks to traverse, ingestion pipelines, etl, etc. that are all factors to keep in mind when considering how soon
```

Reply:
Also, if the network is borked, we'll reliably retry.